### ISSUE TITLE
NuGet.Server 2.14.0 does not show Published package property in Visual Studio UI

### ISSUE THREAD

**Issue by joelverhagen** [NuGet/NuGetGallery#4689]
_**Copied from https://github.com/NuGet/NuGet.Server/pull/23, on behalf of @sean-rose.**_

@joelverhagen  I've run into a problem:  I was particularly interested in getting the published date fix, but now in my testing no published date is being shown at all for packages when browsing via Visual Studio (tested in VS 2013, VS 2015, and VS 2017).

Looking through the history, I believe the cause is in commit 9a2dd68 the following attribute was added to the `ODataPackage` class:

```C#
    [EntityPropertyMapping("Published", SyndicationItemProperty.Published, SyndicationTextContentKind.Plaintext, keepInContent: false)]
```

I'd be happy to submit a pull request to change that to `keepInContent: true` but I'm not familiar enough with the reason for adding the attribute in the first place to know what else that might affect.

**Comment by sean-rose** [comment]
To troubleshoot this I compared the packages Atom feed XML output from NuGet.Server 2.11.2 and 2.14.0.  The 2.14.0 output now includes a `<published>` element within each `<entry>`, but no longer includes a `<d:Published>` element within the entry's `<m:properties>`.  Based on my testing in Visual Studio 2013, 2015, and 2017, Visual Studio relies on the `<d:Published>` element to display the "_Date published_" (or "_Last Published_" in VS 2013) value for a package.

I have tested and verified that changing the `ODataPackage` class's `[EntityPropertyMapping("Published", ... keepInContent: false)]` attribute to `keepInContent: true` resolves this problem such that the `<d:Published>` element is included again and Visual Studio displays the published date for the package.  However, like I said I'm not sure what other impacts that might have.

I'll submit a pull request with that change shortly.  The nice thing is if this is fixed, when combined with the published date change in NuGet/NuGet.Server#23 then issue #3197 will end up getting properly resolved.

**Comment by sean-rose** [comment]
@joelverhagen I went to create the pull request, and the guidelines in the message box say "_Make sure the PR targets the `dev` branch_", but the `dev` branch has diverged drastically from `master`.  So I tested the `dev` branch code, and this published date problem is not present in `dev`, but is that anywhere near ready for release?

If not, would it be OK to submit my pull request targeting `master` to get this fixed in a release?

**Comment by joelverhagen** [comment]
@sean-rose, `dev` is a Web API implementation of NuGet.Server that we hope to release after some stabilization of 2.14.0. If you would like to make this change available soon, feel free to target `master` in this case.

**Comment by joelverhagen** [comment]
Thanks for the fix @sean-rose. I've merged it into master. I hope to release your fix as a `2.14.1` package from `master` next week and also release the Web API changes as a 3.0.0-beta from `dev` (so people can start playing with the new Web API version).

**Comment by joelverhagen** [comment]
Hey @sean-rose, I have published `2.14.1` to NuGet.org.
https://www.nuget.org/packages/NuGet.Server/2.14.1
